src/Wallabag/CoreBundle/Helper/DownloadImages.php

```diff
@@ -146,6 +147,26 @@ public function processSingleImage($entryId, $imagePath, $url, $relativePath = n
 
         $hashImage = hash('crc32', $absolutePath);
         $localPath = $folderPath . '/' . $hashImage . '.' . $ext;
+        $urlPath = $this->wallabagUrl . '/assets/images/' . $relativePath . '/' . $hashImage . '.' . $ext;
+
+        // custom case for SVG (because GD doesn't support SVG)
+        if ('svg' === $ext) {
+            $sanitizer = new Sanitizer();
+            $sanitizer->minify(true);
+            $sanitizer->removeRemoteReferences(true);
+            $cleanSVG = $sanitizer->sanitize((string) $res->getBody());
+
+            // add an extra validation by checking about `<svg `
+            if (false === $cleanSVG || false === strpos($cleanSVG, '<svg ')) {
+                $this->logger->error('DownloadImages: Bad SVG given', ['path' => $imagePath]);
+
+                return false;
+            }
```

Comment:
Shouldn't we put a try-catch here, like for lines 171-177?

Reply:
What other alternative we have, like in the catch from lines 171-177?
Or maybe just a try/catch with a `return false` in the catch?

Reply:
Yes, we could put `return false` in the catch